scm/traverse/repos.go

```diff
@@ -20,10 +20,8 @@ func Repos(ctx context.Context, client *scm.Client) ([]*scm.Repository, error) {
 		if err != nil {
 			return nil, err
 		}
-		for _, src := range result {
-			if src != nil {
-				list = append(list, result...)
```

Comment:
Change `result` to `src` in this line. Intended code was to add the repo if it is not nil